package HackerRank;import java.util.InputMismatchException;import java.util.Scanner;/*** * At a time two types of exception handling */public class ExceptionTryCatch {    public static void main(String[] args) {        Scanner scanner = new Scanner(System.in);        int number1;        int number2;        try {            number1 = scanner.nextInt();            number2 = scanner.nextInt();            if(number1 >= Math.pow(2, 31) - 1 || number2 >= Math.pow(2, 31) - 1){                throw new InputMismatchException();            }            if(number2 == 0){                throw new ArithmeticException("java.lang.ArithmeticException: / by zero");            }            System.out.println(number1 / number2);        }        catch (InputMismatchException inputMismatchException){            System.out.println("java.util.InputMismatchException");        }        catch (ArithmeticException arithmeticException){            System.out.println(arithmeticException.getMessage());        }        scanner.close();    }}